contribution/words.json

```diff
@@ -34,6 +35,7 @@
 		"counting binaries",
 		"counting sheep",
 		"counting the ones and zeros that make your PC function",
+		"creating a pull request",
```

Comment:
Maybe add "merging a pull request" too?

Reply:
good idea!

Reply:
Want me to create a PR?

Reply:
@joycedfan https://github.com/joycedfan/CyberCodeOnline/pull/1

Reply:
Do I have to make a new PR for that or can I edit this PR?

Reply:
@joycedfan Just merge mine.